src/pages/public/about-us-page/company-values/CompanyValues.tsx

```diff
@@ -1,28 +1,51 @@
 import { ABOUT_US_DATA } from '../../../../const/public/about-us-page';
+import { useState, useEffect } from 'react';
 import './CompanyValues.scss';
+import { CustomSwiper } from '../../../../components/public/swiper/CustomSwiper';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { ValueCard } from './components/value-card/ValueCard';
+
+export interface ValueItem {
+    NAME: string;
+    DESCRIPTION: string;
+}
```

Comment:
Intefrace use `camelCase`
```suggestion
export interface ValueItem {
    name: string;
    description: string;
}
```